examples/kafka/kafka-ephemeral-oauth-single-keycloak-authz.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: kafka.strimzi.io/v1beta2
```

Comment:
i think this file should be moved to security examples: https://github.com/strimzi/strimzi-kafka-operator/tree/main/examples/security

Reply:
Yes, we can do that.